.github/ISSUE_TEMPLATE/failing test.md

```diff
@@ -0,0 +1,15 @@
+<!-- Please use this template only for submitting reports about failing tests -->
+
+/kind failing-test
+
+**Which jobs are failing**
+
+**Which test(s) are failing**
+
+**Since when has it been failing**
+
+**Testgrid link**
+
+**Triage**
```

Comment:
what is Triage in this case? Not sure we need this header, I'd have no idea what to put there, and I'm just finishing a release as bug triage lead...

Reply:
> The idea was to provide what is causing the test to fail according to the person filing the issue.

Can this go in `Anything else we need to know` or be renamed as `Why it might be failing` or something like that?

Reply:
Also, should this be Failing/Flaky test or should they be different templates?

Reply:
> Also, should this be Failing/Flaky test or should they be different templates?

I think it could be clubbed together...maybe a commented `/kind flake` mentioning to uncomment it when the test is a flaky one?

As an aside: should this be `area/flake` instead?
/cc @spiffxp

Reply:
That usually causes chaos as is the case with `kind/feature` and `kind/bug` right now in k/k. :/